spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/tracing/zipkin/ZipkinConfigurationsSenderConfigurationTests.java

```diff
@@ -60,15 +60,6 @@ void shouldUseRestTemplateSenderIfUrlConnectionSenderIsNotAvailable() {
 				});
 	}
 
-	@Test
-	void shouldNotSupplyRestTemplateSenderIfNoBuilderIsAvailable() {
```

Comment:
Just to double-check my understanding, I assume you have removed this test as it duplicates `shouldSupplyBeans()`?

Reply:
@wilkinsona Yes, it seems to be a duplicate of `shouldSupplyBeans()`.

Reply:
👍